src/components/users/UserLoginForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import emailValidator from 'email-validator'
+
+import RaisedButton from 'material-ui/RaisedButton'
+
+import TextField from './TextFieldExtended'
```

Comment:
I think it it would be better if you use a more descriptive class name for TextFieldExtended (like `ValidatingTextField`) and also use that name in other classes where you import it to avoid confusion.